lib/reporter.js

```diff
@@ -249,7 +250,7 @@ var CoverageReporter = function (rootConfig, helper, logger, emitter) {
         var remappedCoverageMap = sourceMapStore.transformCoverage(coverageMap).map
 
         // If config.check is defined, check coverage levels for each browser
-        if (config.hasOwnProperty('check') && !checkedCoverage[browser.id]) {
+        if (hasOwnProperty.call(config, 'check') && !checkedCoverage[browser.id]) {
```

Comment:
Is this change separate from the deps update?

Reply:
It is new rule from eslint - https://eslint.org/docs/rules/no-prototype-builtins
that why need to fix.